common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -14,49 +13,47 @@
     </li>
 
     <li class="menu-item u-h js-navigation-account-details">
-        <details>
-            <summary class="menu-item__title"
-                     data-link-name="nav2 : my account">
-                <i class="menu-item__toggle"></i>
-                my account
-            </summary>
+        <button class="menu-item__title js-navigation-toggle"
```

Comment:
I was thinking, that we then could close the list by default, because some of the links aren't working without JS (links are build in JS). Eventhough it's closed then, you have a chance to go to your account and do all desired actions there ..